src/test/rgw/bucket_notification/test_bn.py

```diff
@@ -3128,6 +3128,113 @@ def test_ps_s3_persistent_topic_stats():
     time.sleep(delay)
     http_server.close()
 
+@attr('basic_test')
+def test_ps_s3_persistent_topic_configs():
```

Comment:
please add a similar test for retry count + retry sleep.
you can you the count + sleep time to calculate the sleep time needed for the test itself.